pkg/ocm/clusters.go

```diff
@@ -392,6 +392,23 @@ func (c *Client) getSubscriptionByExternalID(externalID string) (*amv1.Subscript
 	return response.Items().Slice()[0], true, nil
 }
 
+func (c *Client) GetSubscriptionByID(id string) (*amv1.Subscription, bool, error) {
+	query := fmt.Sprintf("cluster_id = '%s'", id)
+	response, err := c.ocm.AccountsMgmt().V1().Subscriptions().List().
```

Comment:
Should we use get :thinking: as we have the subscription id in the cluster spec we can pass it in here? WDYT?

Reply:
done!